### PR TITLE
removed from models import * #fixed 1057

### PR DESCRIPTION
**Pull request by ankit01ojha**
removed the package bodhi.server.models and replaced it with bodhi.server.models.py.

### CODE REVIEW

**Comment by ankit01ojha** [comment]
@trishnaguha : Can you please review this Pull Request :)

File: bodhi/server/models.py [hunk 1]
```diff
@@ -1709,41 +1804,39 @@ def check_requirements(self, session, settings):
         return True, "All checks pass."
 
     def check_karma_thresholds(self, db, agent):
-        """
-        Check if we have reached either karma threshold, call set_request if necessary,
-        and Ignore karma thresholds if the update is locked and raise exception
-        """
-        # Raise Exception if the update is locked
-        if self.locked:
+        """Check if we have reached either karma threshold, and call set_request if necessary"""
```
Comment (trishnaguha) [inline bodhi/server/models.py:1807]: @ankit01ojha Looks like you do not have the recent updated code base. 
Please make sure to rebase upstream code base branch with your local branch.

**Review by trishnaguha — CHANGES_REQUESTED** [review]

**Comment by ankit01ojha** [comment]
@trishnaguha Its ok now :)

**Comment by ankit01ojha** [comment]
@trishnaguha I tried pushing the updated code using "git commit --amend" , but it seems that the updated version is not getting pushed. Can you please tell me how to push the updated version of the code without increasing the number of commits.

**Comment by ankit01ojha** [comment]
@trishnaguha Is this OK now ?

File: bodhi/server/models.py [hunk 2]
```diff
@@ -25,16 +25,8 @@
 from textwrap import wrap
 from datetime import datetime
 from collections import defaultdict
-
-try:
-    # python3
-    from urllib.parse import quote
-except ImportError:
-    from urllib import quote
```
Comment (bowlofeggs) [inline bodhi/server/models.py:33]: PEP-8 states that imports should be grouped into three groups, at the top of the file:

1. Standard library
2. Third party
3. Your package

Since urlib is from the standard library, it should stay here next to other standard library imports (like datetime and collections).

File: bodhi/server/models.py [hunk 3]
```diff
@@ -43,6 +35,8 @@
 from sqlalchemy.orm.properties import RelationshipProperty
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm.exc import NoResultFound
+from sqlalchemy.types import SchemaType, TypeDecorator, Enum
+from sqlalchemy.types import SchemaType, TypeDecorator, Enum
```
Comment (bowlofeggs) [inline bodhi/server/models.py:39]: This line is duplicated.

File: bodhi/server/models.py [hunk 4]
```diff
@@ -158,12 +264,10 @@ def grid_columns(cls):
 
     def update_relationship(self, name, model, data, db):
         """Add items to or remove items from a many-to-many relationship
-
```
Comment (bowlofeggs) [inline bodhi/server/models.py:161]: Let's not make any unrelated changes in this commit, so it can focus on just renaming the module and ensuring that other modules import it correctly.

File: bodhi/server/models.py [hunk 5]
```diff
@@ -605,7 +708,7 @@ class Update(Base):
 
     title = Column(UnicodeText, unique=True, default=None, index=True)
 
-    autokarma = Column(Boolean, default=True, nullable=False)
+    autokarma = Column(Boolean, default=True)
```
Comment (bowlofeggs) [inline bodhi/server/models.py:711]: We do not want this to be nullable.

File: .gitignore [hunk 6]
```diff
@@ -23,3 +23,5 @@ dogpile-cache.dbm*
 html-docs
 masher/
 Vagrantfile
+.python
+*.idea/*
```
Comment (bowlofeggs) [inline .gitignore:27]: This change is OK in general, but let's make it in a separate commit so that this commit remains focused on just renaming the module.
